cfme/tests/services/test_dialog_element_in_catalog.py

```diff
@@ -1136,3 +1138,41 @@ def test_dialog_default_value_selection(appliance, custom_categories, import_dat
         view.fields("network").read() == network and
         view.fields("number_disk").read() == additional_disks
     )
+
+
+@pytest.mark.meta(automates=[1559030])
+@pytest.mark.tier(2)
+@pytest.mark.customer_scenario
+@pytest.mark.parametrize(
+    "import_data", [DatastoreImport("bz_1558926.zip", "bz_1558926", None)], ids=["domain"],)
+@pytest.mark.parametrize("file_name", ["bz_1559030.yml"], ids=["dialog"])
+def test_save_dynamic_multi_drop_down_dialog(appliance, import_datastore, import_dialog,
+                                             import_data):
+    """
+    Bugzilla:
+        1559030
+    Polarion:
+        assignee: nansari
+        casecomponent: Services
+        testtype: functional
+        initialEstimate: 1/4h
+        startsin: 5.10
+    """
+    sd, ele_label = import_dialog
+    navigate_to(sd, "Edit")
+
+    # update dialog element
+    view = appliance.browser.create_view(EditElementView)
+    view.element.edit_element(ele_label)
```

Comment:
Isn't this navigation(navigate_to(sd, "Edit")) helps to reach at this page - EditElementView?

Reply:
No, we have to navigate by EditElementView